app/models/form/sales/questions/mortgageused.rb

```diff
@@ -10,21 +10,24 @@ def initialize(id, hsh, subsection, ownershipsch:)
     @question_number = QUESTION_NUMBER_FROM_YEAR_AND_OWNERSHIP.fetch(form.start_date.year, QUESTION_NUMBER_FROM_YEAR_AND_OWNERSHIP.max_by { |k, _v| k }.last)[ownershipsch]
   end
 
+  def displayed_answer_options(log, _user = nil)
+    if log.outright_sale? && log.saledate && !form.start_year_after_2024?
+      answer_options_without_dont_know
+    elsif log.stairowned == 100 || log.outright_sale?
+      ANSWER_OPTIONS
+    else
+      answer_options_without_dont_know
+    end
+  end
```

Comment:
I know this looks weird with having two branches with the same content but I ended up finding it the most readable.
It's made a bit more complicated by two branches with different outcomes both featuring `:outright_sale?` and trying to combine those two branches I always ended up with some very hard to read negated conditions.